ci/jenkins/test-vmc.sh

```diff
@@ -467,6 +473,46 @@ function deliver_antrea {
     done
 }
 
+function deliver-flow-visibility {
+    echo "====== Pulling and delivering images in flow-visbility.yml to all the Nodes======"
+    FV_IMAGES=($(cat $GIT_CHECKOUT_DIR/build/yamls/flow-visibility.yml | grep image: | awk '{print $(NF)}' | xargs))
+    IPs=($(kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | xargs))
+    old_IFS=$IFS
+    for image in "${FV_IMAGES[@]}"; do
+        docker pull $image
+        IFS=:
+        read repo tag <<< "$image"
+        IFS=$old_IFS
+        echo "Saving" $repo:$tag
+        docker save -o flow-visibility.tar $repo:$tag
+        for i in "${!IPs[@]}"
+        do
+            copy_image flow-visibility.tar $repo ${IPs[$i]} $tag false
+        done
+    done
```

Comment:
is this needed? can't we let each Node pull the image here, since we don't need to retag it?

Reply:
Thanks for pointing it out! Yes, all changes in `test-vmc.sh ` are no longer needed right now since we have all images inside the VMware harbor.